rfcs/0077-stale-issues-amendment.md

```diff
@@ -0,0 +1,135 @@
+
+People with vested interest should see the _stale_ label and be prompted
+to think: "Oh, I need this, too. Damn, it's stale. Let's have a look
+and do something to help out."
+
+An issue that hasn't been interacted with for 60 days or a PR for 90 days
+and in the vast majority of cases deserves this hint out of fariness.
+
+Honesty and transparency are by far more effective than any perceived
+political correctness (judging by the discussions on RFC 51).
+
+# Detailed design
+[design]: #detailed-design
+
+- Mark PRs as stale after 90 days
+- Mark issues as stale after 60 days
+- Keep the policy of never closing either PRs or Issues.
+- Add `2.status: never-stale`, which maintainers can use on long running or umbrella issues.
+
+_Note: non-maintainers are sufficiently unlikely to open truely long-runnig or umbrella 
+issues without a maintainer stepping up to prevent them from getting marked as stale._
+
+# Examples and Interactions
+[examples-and-interactions]: #examples-and-interactions
+
+It should be noted under this section, that it only takes a comment
+to un-stale an issue or PR.
+
+It should be made clear, that stale does **not** mean either of
+the following:
+
+- bad
+- unimportant
+- invalid
+- not useful
+
+Or any similar deminishing interpretations. Stale just means an aggregate
+indcator of individual's choices to not interact.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+With a renewed definition of the word _stale_ and removing an overly emotional
+meaning by virtue of this very PR, there is no reason to adversely interpret
+a bot spelling out the facts.
+
+Regular subscriber might get notiviations they do not find immediatly
+actionable for them personally. However, the bot is inteded to help authors,
+not (silent) subscribers. It is assumed that subscribers wisely choose
+and actively manage how they spread their limmited attention. At any rate,
+their interests should never outperform the interest of the auther (="owner").
+
+At a closer look, increased notification is also a valuable _gain_ in information
+for those subscribers. Imagine: A topic went stale, then there where people moving
+things forward. Then it went stale _again_. If I'm subscribed to a topic of 
+_interest_, a second stale promts me to consider taking action.
+
+Some data: As of now, there are roughly 1750 open issues marked as stale, and
+roughly 450 stale issues were marked as closed. Even older ones. By definition
+of how the stale bot operates, this means the stale bot has failed in adequately
+prompting action. The data suggests, it is _not_ actionable.
+
+There are a variety of possible reasons for this, but if we assume that the bot
+should have prompted the _author_ into "furthering the cause" (whatever that
+might be in a particular context), judging by the 1750 unattended interventions,
+it blatantly failed. Since any such furthering would have left traces the stalebot
+would have picked up in any way.
```

Comment:
Expecting people to do something they may not have time/energy/motivation to is IMHO wrong in an open source project where a majority contributes in their *free* time.

Reply:
The only underlying expectation expressed in this RFC is to clean up after oneselves.

How come, you interpret that way?

Reply:
If removed this section and reworked the interpretation of the underlying available data. Feel free to un resolve if there is anything more to it that should I take into account. Thanks! for raising the issue about this part and helping me to make it clearer.